core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,13 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Closes the Iterator. This should be overridden by the implementing class that has access to <tt>SortedKeyValueIterator source</tt> provided in the
```

Comment:
Simplified version (still nothing about exceptions though): 
Closes any resources that were opened by the <tt>SortedKeyValueIterator</tt>. This method does nothing by default. If the implementing class opens any kind of resources, such as <tt>source</tt> provided in the <tt>init</tt> method, than it should be overridden to close those resources.